cmd/cli/verify_connectivity.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/action"
+)
+
+const verifyConnectivityDescription = `
+This command consists of multiple subcommands related to verifying
+connectivity related configurations.
+`
+
+func newVerifyConnectivityCmd(config *action.Configuration, stdout io.Writer, stderr io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "connectivity",
+		Short: "verify connectivity configurations",
+		Long:  verifyConnectivityDescription,
+		Args:  cobra.NoArgs,
+	}
+	cmd.AddCommand(newVerifyConnectivityPodToPodCmd(config, stdout, stderr))
```

Comment:
> > I'm curious what future commands/subcommands will look like, and if we can take a more kubectl-esque approach.

> > ie: maybe osm verify connectivity --from [<pod>|<kind>/<name>] --to [<pod>|<kind>/<name>] [-n|--namespace <namespace>]

> The immediate scenarios to address are listed [here](https://docs.google.com/document/d/1-HPGNtnch6ahYkcrjPv84UKr7AVeTO4X0Ykb1b2-zGo/edit?usp=sharing).

> Regarding flags, it comes down to preference. I like using positional args for mandatory arguments, but see how using required flags can make the arguments more explicit.

Personally, I prefer using deployment/sts since it allows me to not have to look up & copy/paste pod each time. ie: `kubectl logs deployment/bookbuyer -n bookbuyer`. Having something like that would be a slightly cleaner API.

I think it would also help since it allows us to permute the different variations. ie: To/From
* service
* service identity
* pod
* deployment
* statefulset
* etc

So maybe with that in mind we switch to `osm verify connectivity --from-pod <namespace/pod> --to-service <namespace/service>`.

That loses the kubectl-likeness I was shooting for originally, but allows these permutations

Reply:
I think deployments/sts/service could be a shorthand, but I think the base functionality should allow users to specify a given pod. I've run into several scenarios where one pod in particular has an issue getting XDS updates and I need to exec and and test curls to/from that pod. Furthermore, each OSM workload cert has a CN with a uuid corresponding to the pod name, so that increases the weird scenarios customers could get into.

Again, I think having deployment/sts/service(maybe we should generalize to FQDN?) shorthand is still very useful. I just want to make sure we're not excluding the pod to pod connectivity as the base case.

Reply:
I dropped the `pod-to-pod` subcommand in favor of flags. For now, both `from-pod` and `to-pod` are mandatory till we allow additional permutations, which won't happen anytime soon.